Elements.css

```diff
@@ -1,117 +1,141 @@
 #outside_container {
-  position: fixed;
-  top: 0;
-  left: 0;
-  width: 100%;
-  height: 100%;
-  background: url(background.jpg);
-  z-index: -1;
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    background: url(background.jpg);
+    z-index: -1;
 }
 
 #container {
-  top: 0;
-  left: 0;
-  width: 100%;
-  height: 100%;
-
-  z-index: 1;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    z-index: 1;
+    margin-bottom: 50px;
+    overflow: auto;
 }
 
 .container_2 {
-  top: 0;
-  left: 0;
-  width: 100%;
-  height: 100%;
-
-  z-index: 1;
+    border: 2px solid #000000c5;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    z-index: 1;
+    background: #c0d698c5;
+    overflow: auto;
+    margin-bottom: 50px;
 }
 
 #avatar_container {
-  position: relative;
-  top: 30%;
-  left: 50%;
-  transform: translate(-50%, -70%);
-  width: 50%;
-  height: 20%;
+    position: relative;
+    top: 30%;
+    left: 50%;
+    transform: translate(-50%, -70%);
```

Comment:
Very strange center alignment. You can use _**margin: 0 auto;**_ to align elements at the center horizontally.
And please, replace same wrong alignment in other places.

Reply:
fixed